packages/validator-ajv8/test/validator.test.ts

```diff
@@ -401,26 +452,19 @@ describe("AJV8Validator", () => {
           errorSchema = result.errorSchema;
         });
         it("should return an error list", () => {
-          expect(errors).toHaveLength(2);
-          expect(errors[0].name).toEqual("type");
-          expect(errors[0].property).toEqual(".properties.foo.required");
-          // TODO: This schema path is wrong due to a bug in ajv; change this test when https://github.com/ajv-validator/ajv/issues/512 is fixed.
-          expect(errors[0].schemaPath).toEqual(
-            "#/definitions/stringArray/type"
-          );
-          expect(errors[0].message).toEqual("must be array");
-
-          expect(errors[1].stack).toEqual(
+          expect(errors).toHaveLength(1);
+          expect(errors[0].stack).toEqual(
             "schema is invalid: data/properties/foo/required must be array"
           );
         });
```

Comment:
Breaking change (need to document)--if the schema is invalid, don't attempt to validate the data.

I made this change because otherwise we would have to do something like

```js
try {
	compiledSchema = this.ajv.compile(schema)
	compiledSchema.validate(data)
} catch (e1) {
	try {
		this.ajv.validate(schema, data)
	} catch (e2) {
		compilationError = e2;
	}
}
```

IMO this was getting too complex to support a case that probably shouldn't be happening.